sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+from "customers"
+where "isActive" = 'false';
```

Comment:
`false` is a boolean, so you should just use it without the quotes: 
`where "isActive" = false`